contrib/cirrus/setup.sh

```diff
@@ -19,7 +19,11 @@ msg "************************************************************"
 msg "Setting up runtime environment"
 msg "************************************************************"
 
-dnf -y install make automake gcc gcc-c++ kernel-devel
+dnf -y copr enable rhcontainerbot/podman-next
+dnf -y install make automake gcc gcc-c++ kernel-devel aardvark-dns
```

Comment:
This is fine, but please check c/automation_images if `make`, `automake`, `gcc`, `gcc-c++`, and `kernel-devel` are already installed in the fedora-netavark images.  If so, we don't need to install them here, if not please open a PR to add them over there.

(It's a relatively minor thing, so no need to actually incorporate new images)

Reply:
will do

Reply:
done